Bottom: 826460f014ffae1f2da9b608b972ea4e695686c2
Top:    0a346b072584964eeb4e3a84449a27cd5a0417d9
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-05-06 14:02:35 +0530

Refresh of sched-idle-add-debugging-bits

---

diff --git a/kernel/sched/idle.c b/kernel/sched/idle.c
index 579817ca3fdf..5834273378b6 100644
--- a/kernel/sched/idle.c
+++ b/kernel/sched/idle.c
@@ -233,7 +233,8 @@ static void do_idle(void)
 	int cpu = smp_processor_id();
 	int pm_disabled = atomic_read(&per_cpu(nr_lat_sensitive, cpu));
 
-	WARN_ON(pm_disabled < 0);
+	if (pm_disabled)
+		pr_info("Inconsistent value of nr_lat_sensitive counter\n");
 
 	/*
 	 * If the arch has a polling bit, we maintain an invariant:
